File: 2442-number-of-arithmetic-triplets/2442-number-of-arithmetic-triplets.py
class Solution:
    def arithmeticTriplets(self, nums: List[int], diff: int) -> int:
        count=0
        for i in range(len(nums)):
            for j in range(i,len(nums)):
                for k in range(j,len(nums)):
                    if (nums[j]-nums[i])==(nums[k]-nums[j]) and (nums[j]-nums[i]==diff) and k>j and j>i:
                        count+=1
        return count

File: 2442-number-of-arithmetic-triplets/2442-number-of-arithmetic-triplets.js
/**
 * @param {number[]} nums
 * @param {number} diff
 * @return {number}
 */
var arithmeticTriplets = function(nums, diff) {
    let count=0;
    let L = nums.length
    for (let i=0; i<L; i++) {
        for (let j=i+1; j<L; j++) {
            for (let k=j+1; k<L; k++) {
                if ((nums[j]-nums[i]===diff) && (nums[k]-nums[j]===diff)) {
                    count+=1
                }
            }
        }
    }
    return count
    
};